netconf/message/commit.go

```diff
@@ -62,3 +64,49 @@ func NewCommit(confirmed bool, confirmTimeout int) *Commit {
 	rpc.MessageID = uuid()
 	return &rpc
 }
+
+// NewConfirmedCommit creates a confirmed-commit message with an optional
+// confirm-timeout and persist token.
+//
+// Example confirmed-commit with persist:
+// <rpc message-id="101">
+// ..<commit>
+// ....<confirmed/>
+// ....<confirm-timeout>120</confirm-timeout>
+// ....<persist>IQ,d4668</persist>
+// ..</commit>
+// </rpc>
+func NewConfirmedCommit(confirmTimeout int, persist string) *Commit {
+	var rpc Commit
+	s := ""
+	rpc.Commit.Confirmed = &s
+	if confirmTimeout > 0 {
+		rpc.Commit.ConfirmTimeout = &confirmTimeout
+	}
```

Comment:
If the previous suggestion is applied, you can do this:
```suggestion
	if confirmTimeout != nil && confirmTimeout > 0 {
		rpc.Commit.ConfirmTimeout = confirmTimeout
	}
```